submissions/servers/servers.md

```diff
@@ -0,0 +1,44 @@
+## Factorio Servers <author>oof2win2</author>
+
+Most of us have been at the point of connecting to a Factorio server at least once, to play with friends or just check out builds of someone else. In today's edition of Alt-F4, I will briefly explain the history of multiplayer, and then I will take a deeper dive into explaining *how* multiplayer works. I will be explaining the usage of Fully Deterministic algorithms and Lockstep algorithms amongst other things.
+
+### History of Multiplayer
+
+In October 2014 with `0.11.0`, multiplayer itself was introduced into the game, it was however being worked on since [`0.9.4`](https://www.factorio.com/blog/post/fff-26). This multiplayer was unlike the one you see today, you couldn't easily 'Join Friend' through Steam or use the server browser - you needed to know the exact IP address of the server. When the first multiplayer was released, there were quite a few bugs, such as [this bug](https://forums.factorio.com/viewtopic.php?t=6285), which didn't allow multiplayer games to last more than 20 seconds. It was of course fixed not even 3 hours later in typical Wube fashion. There was also [this bug](https://forums.factorio.com/viewtopic.php?t=6414), which didn't allow more than 3 people to connect at once - unlike [this 500 players multiplayer session](https://www.factorio.com/blog/post/fff-332) nearly 6 years later. A lot of work has been put into multiplayer development for 500 players to be able to connect at once.
```

Comment:
I think you should change all mentions of a Fatorio version like this:
>\`0.11.0\`

To this:

> Factorio 0.11

I wrote it like you in one of my articles, and it was changed by Therenas :P We need to stay consistent between articles.